prometheus/metrics.go

```diff
@@ -21,13 +21,15 @@ var (
 // MetricsQuery is a common struct for ServiceMetricsQuery and NamespaceMetricsQuery
 type MetricsQuery struct {
 	v1.Range
-	Version      string
 	RateInterval string
 	RateFunc     string
 	Filters      []string
 	ByLabelsIn   []string
 	ByLabelsOut  []string
 	IncludeIstio bool
+	Namespace    string
+	Apps         []string
+	Workload     string
```

Comment:
We should document (comment here) what a "Workload" is / represents. I'm still not clear :)

Reply:
It's hard to write a simple and concise comment that doesn't turn into a complicated definition.